cypress/support/hub-commands.ts

```diff
@@ -252,7 +252,7 @@ Cypress.Commands.add('deleteCollectionsInNamespace', (namespaceName: string) =>
 
 Cypress.Commands.add('createHubRole', () => {
   cy.requestPost<Pick<Role, 'name' | 'description' | 'permissions'>, Role>(pulpAPI`/roles/`, {
-    name: `galaxy.e2erole${randomString(4)}`,
+    name: randomHubName('create_hub_role'),
```

Comment:
```suggestion
    name: randomHubName('galaxy.create_hub_role'),
```

(because roles need the `galaxy.` prefix, right?)